js/jquery.js

```diff
@@ -1059,15 +1059,18 @@ $(function() {
 
 	init_multiselects('select.lms-ui-multiselect');
 
-	$(document).on('mouseup', '[data-target-url]',
```

Comment:
Proponuję mouseup zamienić na click

Reply:
Jak usunie się mouseup to przestanie działać środkowy przycisk myszy. Eventy typu 'click' nie obsługują przycisków myszy. Musi to pozostać, bo wielu użytkowników ceni sobie możliwość otwierania linku w nowym oknie poprzez kliknięcie środkowym przyciskiem myszy. Dlatego właśnie zrobiłem 'double click' żeby pogodzić wszystkie możliwe opcje i potrzeby.

Reply:
Rozumiem, a gdyby dblclick zamienić na click?

Reply:
To będzie działało tak jak do tej pory.

Reply:
No, lepszy dwuklik niż to co jest teraz. Aczkolwiek idealnie by było tak jak w git-1.11